### ISSUE TITLE
Game softlocks after introductions

### ISSUE THREAD

**Issue by Nadekooo** [Nadekooo/YTTDFR#21]
Once again, the mission is pretty self-explanatory.

After using the key on the big box and Sou asking to accompany Sara at the end of the introductions, the game softlocks on the "Loading..." screen on yttd.fr/jouer.

There are seemingly no errors in the console nor in the network.

I have no idea what causes this. Maybe it's due to some animations shenanigans...?

... Good luck on your mission, Vince.

**Comment by vinceh121** [comment]
This happens way before, not sure if related.

```
rpg_core.js:9165 
       GET https://yttd.fr/yttd-abb7c76641b9d33f316434428636c7692d539a3c/img/animations/2-FR.rpgmvp 404 (Not Found
```

**Comment by vinceh121** [comment]
Fixed in bbcfd491af1a3fe0ea8e023e9877773be5e676f2
